onionshare_gui/mode/website_mode/__init__.py

```diff
@@ -0,0 +1,281 @@
+# -*- coding: utf-8 -*-
+"""
+OnionShare | https://onionshare.org/
+
+Copyright (C) 2014-2018 Micah Lee <micah@micahflee.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+import os
+import secrets
```

Comment:
'secrets' module is shipped only with Python 3.6+. This means I can't run it on Fedora 29  (v3.5) and probably other distros.

Is there a way we can have backwards compatibility for older Python 3? 

@micahflee achieved this in his other project PassphraseMe by using `random` on older Python. See https://github.com/micahflee/passphraseme/commit/ce9a2005a8a01b6cc57c92d9a827337aef322b86

Having said that, I can't see where `secrets.choice` is actually being used, so maybe it's redundant and can be removed? Perhaps because you use the `web.slug` as the password for HTTP auth?

Reply:
I did a test a while back and then forgot to remove the import.